src/public/media.ts

```diff
@@ -226,22 +228,30 @@ export namespace media {
     maxMediaCount: number;
 
     /**
-     * Additional properties for customization of select media in mobile devices
+     * Additional properties for customization of select media - Image in mobile devices
      */
     imageProps?: ImageProps;
 
+    /**
+     * Additional properties for customization of select media - VideoAndImage in mobile devices
+     */
+    videoAndImageProps?: VideoAndImageProps;
+
     /**
      * Additional properties for audio capture flows.
      */
     audioProps?: AudioProps;
   }
 
   /**
-   *  All properties in ImageProps are optional and have default values in the platform
+   * @private
```

Comment:
Why are we making this private? Aren't these APIs supposed to be public?

Reply:
There are some common parameters between VideoProps and ImageProps, so MediaProps is a base interface to them, especially needed for mediaType = 3 (where can select/capture both Video and Image).

- For example, a case where JS developer mentions CAMER SWITCHER as FALSE in Image mode and TRUE in Video mode. 
- This functionality is not supported by our platform, i.e. to have different configurations of camera in different modes.
- Therefore to avoid confusion for JS developers, we have created a base interface encapsulating common parameters that will restrict the developers from specifying two different values. 
- Hence there is no direct usage of MediaProps that's why marked this as private to hide it from public docs. Feel free to refer to the Contract document attached in the channel post.